rts/Game/PreGame.cpp

```diff
@@ -322,6 +322,8 @@ void CPreGame::StartServer(const std::string& setupscript)
 		sha512::dump_digest(mapChecksum, mapChecksumHex);
 		sha512::dump_digest(modChecksum, modChecksumHex);
 
+		archiveScanner->WriteCache(); // write the cache, useful in case the game loading crashes afterwards
```

Comment:
Is this actually reachable via `--calc-checksum`?

Reply:
Nope something unrelated, but good to have